docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,366 @@
+
+Each package has its own directory, with one subdirectory for each package revision. Each package revision directory contains the set of files necessary to create a consumable package that can be used by a DC/OS Cluster to install the package. For example, our package would look like this:
+
+```
+└── repo/package/T/time-server
+    ├── 0
+    │   ├── config.json
+    │   ├── resource.json
+    │   ├── marathon.json.mustache
+    │   └── package.json
+    └── ...
+```
+
+In our case, since this is the first version of our time-server, we will create the above folder structure with only one revision (with number 0) and create the required empty files. As the versions of your package grow, this number increments by one unit.
+
+***Tip : When reading the schema json files, look for `required` json field to understand what fields are mandatory***
+
+#### config.json
+As the name says, this file is used for any configuration purposes. This is how our config.json would look like:
+
+```
+{
+	"$schema": "http://json-schema.org/schema#",
+	"properties": {
+		"service": {
+			"type": "object",
+			"description": "DC/OS service configuration properties",
+			"properties": {
+				"name": {
+					"description": "Name of this service instance",
+					"type": "string",
+					"default": "time-server"
+				},
+				"cpus": {
+					"description": "CPU shares to allocate to each service instance.",
+					"type": "number",
+					"default": 0.1,
+					"minimum": 0.1
+				},
+				"mem": {
+					"description":  "Memory to allocate to each service instance.",
+					"type": "number",
+					"default": 256.0,
+					"minimum": 128.0
+				}
+			}
+		}
+	}
+}
+```
+
+We have three main properties to be configured. The `name` is the actual name of the service. The cpus and mem are the amount of CPU and Memory required for each service instance. You can read more about the various fields in this field [here](https://github.com/mesosphere/universe#configjson) or can refer to [`repo/meta/schema/config-schema.json`](repo/meta/schema/config-schema.json) for a full fledged definition.
+
+(Note : If you need to add a config property after the merge of the PR and CI has deployed your package, you have to bump your package version and create new package. So be sure to add all the config properties that you need.)
+
+#### resource.json
+This file contains all of the externally hosted resources (E.g. Docker images, HTTP objects and
+images) needed to install the application. It also contains the `cli` section that can be used to allow a package to configure native CLI subcommands for several platforms and architectures.
+
+Below is the resource file that we use for our package. We have provided our earlier published docker-user-name/time-server:part1 image under the docker field here. Note that giving a docker image is optional and we can have other ways to execute the binary (E.g.: The package dcos-enterprise-cli doesn't use a docker image to install the binary.)
```

Comment:
It should be Docker vs docker. Let's do this throughout the file unless you are referring to the `docker` command.